### DISCUSSION TITLE
Suggested fonts for fitting lots of text + nodes on screen? + Color emojis?

### DISCUSSION THREAD

**Discussion by hi2u** [freeplane/freeplane#1207] [General]
**Readable fonts at small size**

- Just wondering if anybody knows of some good fonts to use in Freeplane, that are more easily readable when you're zoomed out a bit, and want to fit lots of stuff on the screen?
- I guess something that is kinda compact, but also bold enough to easily read when it's small.  
  - ...Obviously a bit hard to balance / get right, seeing they kinda contradict each other! :) 

**Emojis**

- Also curious if people have suggested fonts to use if you want to include emojis in your text?  
- Is there any way to get color emojis at all?

**OS**

- I'm on Windows.  
- But can download any fonts that aren't included by default.
- Although if you have any suggestions for fonts that can't be used on Windows, doesn't hurt to mention, maybe useful to others.

**Comment by kenfhill84083** [comment]
> **Readable fonts at small size**
> 
>     * Just wondering if anybody knows of some good fonts to use in Freeplane, that are more easily readable when you're zoomed out a bit, and want to fit lots of stuff on the screen?
> 

I am also on windows (11).  My favorite font is Segoe UI.  Crisp and clear.  My default font size in maps is 10pt and I can still read pretty well at 33% zoom.

Of course monitor resolutions differ and fonts are a bit toMAYto / toMAWto per individual taste.

**Comment by euu2021** [comment]
I try to get maximum compactness, in order to fit as much information as possible on the screen. In terms of fonts, I suggest that you guys try Dialog, on Windows 11. It has a good readability and is more compact than other fonts. 

Notice how the exact same amount of text takes less space:

![image](https://github.com/freeplane/freeplane/assets/77707706/27f5a133-50c9-4a19-a30b-55b73c5b4ef6)

**Reply by kenfhill84083** [reply]
Thanks for the comparison.  Dialog does seem to be a nice font.  Maybe I am biased, but the Segoe UI seems sharper at the screenshot's size and resolution.

I can see there is more space above Segoe UI text (see image).  It appears Dialog is a bit thicker (the node with "ADI" at the end has less trailing space).  Could you filter on a couple of comparison nodes, hiding ancestors and descendants - so they are close together?  Then zoom.  Then take the screenshot?   (Although it may not be worthwhile to spend any additional time.)

In my 2nd screenshot of your screenshot, I like Segoe's "C" and "t" and "r" better - they are easier for me to discern.  That might be a factor if viewed zoomed out.  (But maybe screenshot resolution could be at play here.)

Font preference is very subjective.  As is a trade off between readability and maximum compactness.

I wanted to do a similar comparison for evaluation here - but I do not have Dialog with my install of Windows 11.  Is it something that tagged along with an app?  Or something you downloaded?  If so, where?  I could not find it online (my searches found hits for "font dialog box" almost exclusively.  Nothing I could find in "web fonts".  (I found a Linotype Dialog but it is expensive and the "g" is different.)  So this option, for me, may be moot.

Thanks.

![freeplane _ font comparison - dialog v segoe ui](https://github.com/freeplane/freeplane/assets/6380301/2844563d-8be5-4a9c-85d8-675d8b786e0a)
![freeplane _ font comparison - dialog v segoe ui 2](https://github.com/freeplane/freeplane/assets/6380301/13b23edc-13d9-47f2-920d-de5bc6289b00)

**Reply by euu2021** [reply]
I checked again all the fonts in my computer and found a new favorite: Arial Narrow. It has the same height as other fonts, but it's more narrow. The readability suffers a bit, but it worked for me.
Comparison with Segoe UI:
![image](https://github.com/freeplane/freeplane/assets/77707706/e5f06a3e-5793-4ca8-9546-3f31bb008387)

The height of the letters is exactly the same:
![image](https://github.com/freeplane/freeplane/assets/77707706/79d8a88d-97f3-415c-9aa6-d9bda03eb30c)

About the Dialog font I don't know where it came from. But, I compared it with standard Arial and noticed that Arial fits even better than Dialog into the node. So, I think Dialog can be discarded. 

![image](https://github.com/freeplane/freeplane/assets/77707706/9f561fee-0dbb-4a69-b1c4-40a5b93f90b4)

Doing a visual comparison with the letters very zoomed out, I think that Segoe UI has a slightly better readability. So, my conclusion is:

-readability: Segoe UI
-extreme compression: Arial Narrow
-middle ground: Arial

**Reply by kenfhill84083** [reply]
You might want to compare Arial Narrow with Liberation Sans Narrow.  Supposedly the metrics makes LSN fit exactly the same space as AN in documents.  That is verified in the screenshots - the "quick brown fox..." line lengths are exact matches.

LSN seems to have a bit thinner strokes - especially where oval shapes meet straight shapes (b, p, d, g, q...) and where letters have "humps" (u, m, n, h).  And I like the way "e" has a more open gap.

I like the way LSN sits a bit higher in a fork shape node - seems more readable to me.  In rectangle shape nodes AN seems more centered but not by much.

In the both-sides map, neither font made for extra height.

My eyeball test says LSN has a slight readability edge over AN.  I may switch to LSN.

I have read that the Liberation font family is the default in Libreoffice (which matches my experience) so maybe that is how it got in my fonts list.

![freeplane _ fonts - rectangle shape - arial narrow (top) , liberation sans narrow (bottom)](https://github.com/freeplane/freeplane/assets/6380301/2861b9bc-c7fc-4f57-b953-cf78fd906750)
![freeplane _ fonts - fork shape - arial narrow (top) , liberation sans narrow (bottom)](https://github.com/freeplane/freeplane/assets/6380301/488588c5-9a15-4508-851a-aaed8a2e23bf)
![freeplane _ fonts - map height - arial narrow (right) , liberation sans narrow (L)](https://github.com/freeplane/freeplane/assets/6380301/0dfe74ba-afd6-40ab-a36f-c3281c4b3bae)

**Reply by kenfhill84083** [reply]
I retract part of the previous.  When I made the 1st level nodes ("liberation sans N.") to actually be LSN font, the left side gained height.  

Not by much but you probably will want to stick to arial narrow.

![freeplane _ fonts - map height - arial narrow (right) , liberation sans narrow (L) - after font change](https://github.com/freeplane/freeplane/assets/6380301/19ebaf50-b750-4940-a537-18710106dcd6)

**Reply by kenfhill84083** [reply]
One more comparison.  The Liberation Sans Narrow text seems larger.  Maybe this is an optical illusion tricking my brain.  Or  my brain is tricking itself by confirmation bias.  Maybe the thinner strokes make it seem taller than the more squat Arial Narrow.  Maybe it is time to turn off the computer.

![freeplane _ fonts - from real map](https://github.com/freeplane/freeplane/assets/6380301/cde1df46-d8ad-4ff4-9487-40a9db8f0971)

**Comment by euu2021** [comment]
LSN has a strong vertical shift, which makes it an interesting option for a fork node, as you said. On a node with borders, it can get dangerously close to the top border:

![image](https://github.com/freeplane/freeplane/assets/77707706/864d3cea-117a-4ced-970a-07d58a5ce893)


In terms of height, they are the same:


![image](https://github.com/freeplane/freeplane/assets/77707706/159e9aee-0727-4c7a-836d-7445d674bd30)

As you pointed, LSN has a more open "e", which can make it more readable.
